conu/docker/core.py

```diff
@@ -168,3 +168,23 @@ def read_file(self, file_path):
         # since run_cmd does split, we need to wrap like this because the command
         # is actually being wrapped in bash -c -- time for a drink
         return self.execute(["cat", file_path], shell=False)
+
+    def check_file_exists(self, filename):
+        """
+        Function checks if file exists in container
+
+        :param filename: Specify filename to check in container
+        :return: True if all files exists
+                 False if at least one does not exist
+        """
+        found = True
```

Comment:
I absolutely don't understand why the implementation is sooooooo complicated, why it can't be a variant of:
```
try:
    self.execute(["ls", "-d", path], shell=False)
except CalledProc... as ex:
    logger.error(...)
    return False
else:
    return True
```

This implementation isn't ideal too (what if there is no `ls` in the container?), but there are plans to create a dedicated `File`-like interface.